dartfn/pubspec.yaml

```diff
@@ -2,32 +2,32 @@ name: dartfn
 description: >
   A tools package for managing FaaS (Function as a service) portable Dart functions projects
 # After changing the version, run `pub run build_runner build`.
-version: 0.4.0
+version: 0.4.1-dev
 homepage: https://github.com/GoogleCloudPlatform/functions-framework-dart
 
 environment:
-  sdk: '>=2.10.0 <3.0.0'
+  sdk: '>=2.11.99 <3.0.0'
```

Comment:
LGTM, just curious about this (why not 2.12.0? Another teachable moment! :))

Reply:
because staeghand is not null safe yet.

Cannot opt into null safety until all of your runtime dependencies do, too – or you'll live in a sea of errors

Reply:
Gotcha, I did see that yesterday. Clever! Thanks.